core/src/main/java/org/apache/iceberg/StaticTableOperations.java

```diff
@@ -32,25 +34,36 @@
   private TableMetadata staticMetadata;
   private final String metadataFileLocation;
   private final FileIO io;
+  private final EncryptionManager encryption;
   private final LocationProvider locationProvider;
 
   /**
-   * Creates a StaticTableOperations tied to a specific static version of the TableMetadata
+   * @deprecated please use {@link #StaticTableOperations(String, FileIO, EncryptionManager)}
```

Comment:
Do we need to deprecate it? To provide a method with a plainText EncryptionManager looks harmless.